meetings/working-groups/expressions-statements/ES-2022-11-30.md

```diff
@@ -0,0 +1,46 @@
+# Bridging expressions and statements working group meeting - Nov 30, 2022
+
+Our agenda was to review proposal issues and make an initial decision on whether or not we wanted to pursue the feature in the short term. We organized the discussion around feature areas and scenarios.
+
+## Making things that are currently statements exist as expressions
+
+- [#973](https://github.com/dotnet/csharplang/issues/973) - **Declaration expressions**
+  This would enable syntax like `(var x = 1).ToString()`.
+  > Resolution: We felt this is a feature will work on in the near term.
```

Comment:
My recollection is that this was not one we decided for near term. What do other folks recall? Since this is not in your summary, is there a "not" missing?

Reply:
Agreed Kathleen.